### ISSUE TITLE
OSGi support

### ISSUE THREAD

**Issue by dschmitz** [scala/pickling#297]
Please add OSGi support to your library by generating the appropriate MANIFEST.MF which can be done with sbt-osgi.

**Comment by hseeberger** [comment]
OMG!

**Comment by eed3si9n** [comment]
Did @hseeberger volunteer for a pull request?

**Comment by hseeberger** [comment]
No, I'd rather volunteer for closing this issue with "won't fix".
